dataland-csvconverter/src/main/kotlin/org/dataland/csvconverter/csv/CompanyInformationCsvParser.kt

```diff
@@ -32,38 +32,40 @@ class CompanyInformationCsvParser {
      */
     fun buildCompanyInformation(row: Map<String, String>): CompanyInformation {
         return CompanyInformation(
-            companyName = companyInformationColumnMapping.getCsvValue("companyName", row),
-            headquarters = companyInformationColumnMapping.getCsvValue("headquarters", row),
-            sector = companyInformationColumnMapping.getCsvValue("sector", row),
+            companyName = companyInformationColumnMapping.getCsvValue("companyName", row)!!,
+            headquarters = companyInformationColumnMapping.getCsvValue("headquarters", row)!!,
+            sector = companyInformationColumnMapping.getCsvValue("sector", row) ?: "Missing Sector",
```

Comment:
"Missing Sector" was added as a workaround, since the current data contains entries where the sector is empty. Should be removed and 'sector' possibly made optional, when implementing DALA-528

Reply:
Changed in the data file and uploaded file to wiki